docs/UserGuide.md

```diff
@@ -110,12 +110,12 @@ Format: `delete_student INDEX` <br>
 Examples:
 * `delete_student 2` deletes the 2nd student in the address book
 
-### Listing all students' emails: `emails`
-Displays concatenated string of students' emails, separated by `;`. Useful for sending mass emails to students.
+### Listing (filtered) students' emails: `emails`
```

Comment:
Hahaha, wait I placed them in brackets because I'm not sure which term we should go with 😄

Reply:
Haha LMAO, but I think its fine tou, it allows the reader to understand that its filtered, and can be the full list also.

Reply:
I'm not sure if the bracket filtered feels abit out of place. Maybe something like `Listing students' emails based on current list`? Hahaha, maybe someone else can phrase it better!

Reply:
Alright, updated to Listing students' emails based on current list instead and also updated the TOC